src/internal/transport/AbstractRequestProcessor.ts

```diff
@@ -89,14 +98,18 @@ export abstract class AbstractRequestProcessor implements RequestProcessor {
                 },
                 secrets: event.secrets,
             };
-            const ctx: HandlerContext & AutomationContextAware = {
+            const ctx: any = {
```

Comment:
Same question, is it because AutomationContextAware has no lifecycle? Should it?

Reply:
Same as above.